src/main/java/com/google/swarm/tokenization/avro/AvroReaderSplittableDoFn.java

```diff
@@ -42,8 +42,8 @@ public class AvroReaderSplittableDoFn
     extends DoFn<KV<String, ReadableFile>, KV<String, GenericRecord>> {
 
   public static final Logger LOG = LoggerFactory.getLogger(AvroReaderSplittableDoFn.class);
-  private final Counter numberOfAvroRecordsIngested =
-      Metrics.counter(AvroReaderSplittableDoFn.class, "numberOfAvroRecordsIngested");
+  private final Counter numberOfRowsRead =
```

Comment:
Why is this being renamed?

Reply:
Want to aggregate the rows read for all file types, so that it is easier to compare with rows inspected/deidentified and detect failures if any.

Reply:
The counter are reported differently based on the transforms they are declared in. But in monitoring we can find the total sum based on the name of the counter.